fixieai/cli/session/commands.py

```diff
@@ -18,11 +18,12 @@ def list_sessions(ctx):
 
 
 @session.command("new", help="Creates a new session and opens it.")
+@click.option("-a", "--agent", required=False)
```

Comment:
Please add a callback that verifies agent exists and exists gracefully if not. See [auth_force_callback as an example](https://github.com/fixie-ai/fixie-sdk/blob/6691d392c500081d8b46ea24a8ed5fe71e5d022a/fixieai/cli/auth/commands.py#L8).

Reply:
ok, will give it a shot